vagrant/vagrantXterm.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+xterm -e 'vagrant ssh' &
```

Comment:
Will extend code (or consider a pull request) if this surfaces as an issue in practice.